fuse_core/include/fuse_core/ceres_options.h

```diff
@@ -190,7 +190,7 @@ CERES_OPTION_STRING_DEFINITIONS(VisibilityClusteringType)
  * @param[in] default_value - A default value to use if the provided parameter name does not exist
  * @return The loaded (or default) value
  */
-template <class T>
+template <class T>  // TODO(CH3): Replace nh with a node
```

Comment:
I'd recommend it takes a `NodeLoggingInterface` and `NodeParamsInterface` with another convenience method that gets those off of "Node like" objects. That will allow it to work with Lifecycle nodes too.

Reply:
Comment updated https://github.com/locusrobotics/fuse/pull/279/commits/771e4f4b228a1bb1c4dea24ad07fbcd46502ea71

Pending merge of https://github.com/ros2/rclcpp/pull/2041 , but if we expect that to take too long I can duplicate an implementation in fuse_core...

Reply:
Reverting the comment update since it turns out we do have to store a SharedPtr to a node after all (for now at least)